docs/advanced-features/using-mdx.md

```diff
@@ -0,0 +1,166 @@
+---
+description: Learn how to use @next/mdx in your Next.js project.
+---
+
+# Using MDX with Next.js
+
+Next.js supports MDX through a number of different means, this page will outline some of the ways you can begin integrating MDX into your Next.js project.
+
+## What is MDX?
+
+MDX is a superset of markdown that lets you write JSX directly in your markdown files. It is a powerful way to add dynamic interactivity, and embed components within your content, helping you to bring your pages to life.
```

Comment:
This feels like it could be the intro paragraph. Also, is Markdown capitalized as a "product"? Not sure.

Reply:
I like to lowercase it. `markdown` feels more like a family of related things, like `markup`. But `CommonMark` or `JavaScript` feel like something that has a committee that owns it behind it!